config/canonical.json

```
@@ -28715,13 +28717,22 @@
   },
   "shop/stationery|Office Max": {
     "count": 115,
-    "match": ["shop/stationery|OfficeMax"],
     "tags": {
-      "brand": "Office Max",
       "name": "Office Max",
       "shop": "stationery"
     }
   },
+  "shop/stationery|OfficeMax": {
+    "count": 115,
+    "match": ["shop/stationery|OfficeMax"],
```

I think what you're trying to do here is match "Office Max" with a space..
(the `match` property contains the alternative names that we don't prefer)

```suggestion
    "match": ["shop/stationery|Office Max"],
```